nucliadb_node/src/shards/shard_reader.rs

```diff
@@ -345,7 +347,21 @@ impl ShardReader {
                         shard_id: String::default(), // REVIEW: really?
```

Comment:
Since you are modifying this part of the code, do you mind removing this line? Is not necessary since we use `..Default::default()` at the end.